README.md

```diff
@@ -53,6 +53,15 @@ keys:
   This specifies the type of pool to manage.
   Valid values for `type`: `lvm`.
 
+- `shared`
+
+  If set to `true`, the role creates or manages a shared volume group. Requires lvmlockd and
+  dlm services configured and running.
+  Default: `false`
```

Comment:
```suggestion

  Default: `false`

```
Adding new lines, otherwise everything is printed on one line.

Reply:
Fixed.